workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/derived_variables.py

```diff
@@ -131,28 +131,44 @@ def _column_dq2(ds: xr.Dataset) -> xr.DataArray:
 
 def _column_dqu(ds: xr.Dataset) -> xr.DataArray:
     if "column_integrated_dQu" in ds:
-        column_dqu = SECONDS_PER_DAY * ds.column_integrated_dQu
+        warnings.warn(
+            "'column_integrated_dQu' is a deprecated variable name. "
+            "It will not be supported in future versions of fv3net. Use "
+            "'column_integrated_dQu_stress' (units of Pa) instead.",
+            DeprecationWarning,
+        )
+        # convert from m/s/s to Pa by multiplying by surface pressure divided by gravity
+        column_dqu = 100000 / 9.8065 * ds.column_integrated_dQu
```

Comment:
need to use a reference surface pressure here, since don't have the actual (lat-lon dependent) surface pressure available. But since we are deprecating this version of the diagnostic, I think it's okay.

Reply:
I guess over Tibet this might be a little weird but yeah not sure how to do that without a lot of work to get the varying pressure in here